### ISSUE TITLE
Update tidyall setup to work properly with dumped schema classes

### ISSUE THREAD

**Issue by adherzog** [iftechfoundation/ifcomp#93]

**Comment by adherzog** [comment]
@jmacdotorg Once the accompanying PR (#103) is merged into dev, this seems like a good time to run tidyall on the entire codebase. There's only one other active branch and it's front-end only, so it wouldn't be affected by the large changeset. That said, devs may have other work in progress they haven't pushed up yet, and this would likely cause conflicts/headaches.

**Comment by jmacdotorg** [comment]
So noted -- I'll make a it a personal near-future task, thanks.
